macros/users_table.sql

```diff
@@ -68,7 +70,61 @@ where
 select
     * except (rn)
 from
-    users_ordered
+    users_ordering
+where
+    rn = 1
+{% endmacro %}
+
+{% macro databricks__users_table(user_cols, user_keys, user_types) %}
```

Comment:
I think maybe that's the case normally but because I'm going directly from the schema content it isn't? Snowflake doesn't have any transformation on the schema field names, databricks and BQ are converted from camelCase to snake_case? (because snowflake just stores the raw json I think)?

Reply:
You are right, the source is the schema itself and that is before any transformer/loader script making changes to it. It should be camelCase everywhere then. But the question is still the camelCase for Snowflake that needs changing: 
<img width="164" alt="image" src="https://user-images.githubusercontent.com/95634439/203998891-9f8557f3-e090-4a0c-a7d1-db7ddffc4445.png">